.travis.yml

```diff
@@ -20,9 +20,10 @@ script:
   - "ansible-playbook -i tests/inventory tests/$PLAYBOOK --connection=local --sudo"
 
   # Run the role/playbook again, checking to make sure it's idempotent.
+  # Changed is 1 as the packages database will update
   - >
     ansible-playbook -i tests/inventory tests/$PLAYBOOK --connection=local --sudo
-    | grep -q 'changed=0.*failed=0'
+    | grep -q 'changed=1.*failed=0'
```

Comment:
@debo I've got an output log from running the playbook subsequent times - the verbose output from that step is:

```
TASK [role-php : Ensure packages database is up to date] ***********************
task path: /app/role-php/tasks/main.yml:6
Using module file /usr/lib/python2.7/dist-packages/ansible/modules/core/packaging/os/apt.py
<localhost> ESTABLISH LOCAL CONNECTION FOR USER: root
<localhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo ~/.ansible/tmp/ansible-tmp-1485812052.47-124121219652155 `" && echo ansible-tmp-1485812052.47-124121219652155="` echo ~/.ansible/tmp/ansible-tmp-1485812052.47-124121219652155 `" ) && sleep 0'
<localhost> PUT /tmp/tmpxXT2dB TO /root/.ansible/tmp/ansible-tmp-1485812052.47-124121219652155/apt.py
<localhost> EXEC /bin/sh -c 'chmod u+x /root/.ansible/tmp/ansible-tmp-1485812052.47-124121219652155/ /root/.ansible/tmp/ansible-tmp-1485812052.47-124121219652155/apt.py && sleep 0'
<localhost> EXEC /bin/sh -c '/usr/bin/python /root/.ansible/tmp/ansible-tmp-1485812052.47-124121219652155/apt.py; rm -rf "/root/.ansible/tmp/ansible-tmp-1485812052.47-124121219652155/" > /dev/null 2>&1 && sleep 0'
changed: [localhost] => {
    "cache_update_time": 1485811820, 
    "cache_updated": true, 
    "changed": true, 
    "invocation": {
        "module_args": {
            "allow_unauthenticated": false, 
            "autoremove": false, 
            "cache_valid_time": 0, 
            "deb": null, 
            "default_release": null, 
            "dpkg_options": "force-confdef,force-confold", 
            "force": false, 
            "install_recommends": null, 
            "only_upgrade": false, 
            "package": null, 
            "purge": false, 
            "state": "present", 
            "update_cache": true, 
            "upgrade": null
        }, 
        "module_name": "apt"
    }
}
```